### ISSUE TITLE
Per-person "likes" feed

### ISSUE THREAD

**Issue by scripting** [scripting/feedlandSupport#111]
Now each of us has our own feed for our Likes.

http://data.feedland.org/likes/davewiner.xml

That's mine. I think you can figure out where yours is.

You have to like something before it's created though.

The universal one is still there, not sure if it'll still be useful. I always wanted to know who Liked something. This approach makes that really clear.

**Comment by scripting** [comment]
@akaKenSmith — it’s right up your alley. 😀

**Comment by scripting** [comment]
To prove it can be done, I subscribed to my personal Likes feed in Feedly.

<img width="669" alt="image" src="https://user-images.githubusercontent.com/1686843/198393135-a600062d-f09a-4eb5-858c-b5a93137e809.png">

**Comment by gwthompson** [comment]
I've realized this morning that my likes feed has also become my "save for later" feed which is really cool! In the past I would save articles out of NetNewsWire to Pocket which I have used as my "save for later" tool for many years. But now I can stay in Feedland and use my likes feed as my new Pocket. Very Exciting! 

Going to keep on digging!

**Comment by scripting** [comment]
Yeah. Think in terms of feeds we can create. We have this great growing database of feed items. 

It took me about an hour to add that feature yesterday. It'll be even easier when we have the Drummer scripting interface.

**Comment by troutcolor** [comment]
Hi,
I presumed my url was http://data.feedland.org/likes/johnjohnston.xml
But I got a 404 message. 
I took a screenshot and started to write this. 
Then I though I might add another like just in case. I did that, refreshed the page and a whole pile of likes appeared.

Though this might be useful if anyone else sees this.

Like @gwthompson I am thinking of my likes feed as bookmarks/read later.

There was some interesting discussions of likes vs bookmarks in the early days of micro.blog Manton doesn't have public likes but has private bookmarks. I think Feedland is in a different position though. It is unlikely (sic) that likes will become some sort of fame marker as the likes are going on items from feeds rather than on a particular Feedlander. 

Since it is RSS I can also show my latest likes on my WordPress Blog. 

Cheers

John

**Comment by akaKenSmith** [comment]
My experience was like what @troutcolor reported -- error message at first, but when I had a fresh Like in place it went right through as a feed that I could follow.

**Comment by scripting** [comment]
It says it works this way in the announcement at the top of the thread. 

<img width="559" alt="image" src="https://user-images.githubusercontent.com/1686843/198840997-720f4d92-b112-40d3-8973-a3cd3b5386b3.png">

**Comment by scripting** [comment]
BTW, the reason this is important now is that testing the new version is going to require careful reading of the testing docs, so we cover the whole UI of the product before unleashing it on users we don't communicate with. 

When reporting an error, it's really important to review the docs to be sure you correctly understand what's going on. 

This one was simple, the next one will be the opposite of simple. But it's still totally worth doing. A FeedLand that is useful without logging in. But it could also kill the product if there are some issues that should have been addressed before opening it up. 

Please listen, and thanks. ;-)

**Comment by scotthansonde** [comment]
I confess, I understood this incorrectly at first as well. I had liked something last week so I thought the feed would have been created. When it wasn’t, I figured I had to like something new so I did.

**Comment by scripting** [comment]
@scotthansonde -- thanks for that. I try to write the stuff as clearly as possible and anticipate every possible misunderstanding. none of this is personal. think of me as a coach, and we're something like a team, and i'm saying that in an upcoming game if we want to win we're going to have to do this one thing very well or we fail. we're all adults, smart, care, and want this to work, i hope. ;-)
